### ISSUE TITLE
Any plan to shorten the user path to generate questions?

### ISSUE THREAD

**Issue by jungler233** [ECuiDev/obsidian-quiz-generator#21]
now 5 steps are necessary to start register a quiz for a single file: 1) click the button on the left 2) click file selector 3)select a file 4) quit the selector 5) select generator. The user path is too lengthy. Do you plan to streamline this process to 1click or 2clicks maximum?

**Comment by ECuiDev** [comment]
I do have some ideas for this. For example, right-clicking a note or folder to select a generate quiz menu item. I might include this with the next release actually.

**Comment by LeonelRFF** [comment]
Related to the fact of folders, a future feature would be notes linked to said note, that is:

Note A
[[link to note A1]]
[[link to note A2]]

So that the backlinks are included as outlinks that are in the notes.

The [copilot ](https://github.com/logancyang/obsidian-copilot) working on something similar to this, some context references:

- https://github.com/logancyang/obsidian-copilot/issues/326
- https://github.com/logancyang/obsidian-copilot/issues/257
- https://github.com/logancyang/obsidian-copilot/issues/331
- https://github.com/logancyang/obsidian-copilot/issues/648
- https://github.com/logancyang/obsidian-copilot/issues/682
- https://github.com/logancyang/obsidian-copilot/issues/662
- https://github.com/logancyang/obsidian-copilot/issues/717

**Comment by ECuiDev** [comment]
Yup, that's exactly what I had in mind. I'm actually going to move development of the Note Links feature ahead  now that I think about it. Thanks for those links as well :]
